app/Http/Controllers/HomeController.php

```diff
@@ -68,4 +69,61 @@ public function getPicture($filename){
 
         return $response;
     }
+
+    /* Report bug */
+    public function indexReportBug()
+    {
+        return view('report_bug');
+    }
+
+    public function reportBug(Request $request)
+    {
+        $user = Auth::user();
+        if(Auth::user()){
+            $username = htmlspecialchars(stripslashes(Auth::user()->name));
+        } else {
+            $username = 'an unauthenticated user';
+        }
+
+        //personal auth token from your github.com account
+        $token = env('GITHUB_AUTH_TOKEN');
```

Comment:
Could you please add this as an optional step to the contribution guide with the permissions which should be added to the token?

Reply:
I don't see why anyone would use this. We need this only for the production. But if you want I can add.
I don't know the proper permissions, I checked all....

Reply:
> I don't see why anyone would use this.

I would like to use this when upgrading prod :) Also, if anyone wants to test or improve this later.

> I don't know the proper permissions, I checked all...

This will definitely not go into production.